image_compare/test_spec.js

```diff
@@ -0,0 +1,207 @@
+      { file: 'beginEndShape-2dtriangles1.pde', tolerance: 0.008 },
+      { file: 'beginEndShape-2dtriangles2.pde', tolerance: 0.005 },
+      { file: 'beginEndShape-2dtrianglestrip1.pde', tolerance: 0.009 },
+      { file: 'beginEndShape-2dtrianglestrip2.pde', tolerance: 0.008 },
+      { file: 'beginEndShape-2dvertex.pde', tolerance: 0.005 }
+    ]},
+    
+    { suite: 'bezier', tests: [
+      { file: 'bezier.pde', tolerance: 0.005 },
+      { file: 'bezier2.pde', tolerance: 0.005 },
+      { file: 'bezier-ellipse.pde', tolerance: 0.005 }
+    ]},
+
+    { suite: 'blend', tests: [
+      { file: 'blend-add.pde', tolerance: 0.015 },
+      { file: 'blend-blend.pde', tolerance: 0.015 },
+      { file: 'blend-burn.pde', tolerance: 0.015 },
+      { file: 'blend-darkest.pde', tolerance: 0.015 },
+      { file: 'blend-difference.pde', tolerance: 0.015 },
+      { file: 'blend-dodge.pde', tolerance: 0.015 },
+      { file: 'blend-exclusion.pde', tolerance: 0.015 },
+      { file: 'blend-hard_light.pde', tolerance: 0.015 },
+      { file: 'blend-lightest.pde', tolerance: 0.015 },
+      { file: 'blend-multiply.pde', tolerance: 0.015 },
+      { file: 'blend-overlay.pde', tolerance: 0.015 },
+      { file: 'blend-screen.pde', tolerance: 0.015 },
+      { file: 'blend-soft_light.pde', tolerance: 0.015 },
+      // TODO: SUBTRACT blend mode is not defined 
+      { file: 'blend-subtract.pde', tolerance: 0.015, skip: true }
+    ]},
+    
+    // TODO: implement blendColor
+    { suite: 'blendColor', tests: [
+      { file: 'blendcolor-add.pde', tolerance: 0.015, skip: true },
+      { file: 'blendcolor-blend.pde', tolerance: 0.015, skip: true },
+      { file: 'blendcolor-burn.pde', tolerance: 0.015, skip: true },
+      { file: 'blendcolor-darkest.pde', tolerance: 0.015, skip: true },
+      { file: 'blendcolor-difference.pde', tolerance: 0.015, skip: true },
+      { file: 'blendcolor-dodge.pde', tolerance: 0.015, skip: true },
+      { file: 'blendcolor-exclusion.pde', tolerance: 0.015, skip: true },
+      { file: 'blendcolor-hard_light.pde', tolerance: 0.015, skip: true },
+      { file: 'blendcolor-lightest.pde', tolerance: 0.015, skip: true },
+      { file: 'blendcolor-multiply.pde', tolerance: 0.015, skip: true },
+      { file: 'blendcolor-overlay.pde', tolerance: 0.015, skip: true },
+      { file: 'blendcolor-screen.pde', tolerance: 0.015, skip: true },
+      { file: 'blendcolor-soft_light.pde', tolerance: 0.015, skip: true },
+      { file: 'blendcolor-subtract.pde', tolerance: 0.015, skip: true }
+    ]},
+    
+    { file: 'blur.pde', tolerance: 0.015 },
+    { file: 'bounce.pde', tolerance: 0.015, timeout: 1000 },
+    // uses "class" keyword
+    { file: 'bouncy-bubbles.pde', tolerance: 0.015, skip: true },
+    { file: 'brightness.pde', tolerance: 0.015 },
+    { file: 'brightness-2.pde', tolerance: 0.015 },
+    // TODO: randomSeed algorithm differs from processing-js
+    { file: 'brownian.pde', tolerance: 0.015 }
```

Comment:
I ran this test a couple of times and it outputs the same result each time which is what ```randomSeed``` should be doing, it's just the random numbers are different from processing-js (which I believe re-implements the same algorithm as the original Processing).